pkg/kv/kvserver/store_snapshot.go

```diff
@@ -970,9 +974,9 @@ func sendSnapshot(
 		snap,
 		to,
 		durSent.Seconds(),
-		humanizeutil.IBytes(int64(float64(numBytesSent)/durSent.Seconds())),
+		redact.Safe(humanizeutil.IBytes(int64(float64(numBytesSent)/durSent.Seconds()))),
```

Comment:
if we make `IBytes` return a `SafeString` as we could, I assume the diff blows up wildly, right?

Reply:
Yeah, started off doing that, but it got pretty involved. I'll submit a follow-up PR.